OpenRA.Mods.Common/Traits/Render/WithMoveAnimation.cs

```diff
@@ -18,6 +19,9 @@ public class WithMoveAnimationInfo : ITraitInfo, Requires<WithSpriteBodyInfo>, R
 		[Desc("Displayed while moving.")]
 		[SequenceReference] public readonly string MoveSequence = "move";
 
+		[Desc("Which sprite body to modify.")]
+		public readonly string AssignedBodyName = "body";
```

Comment:
Could we make this a `readonly string[] BodyNames` to match what we do for armaments?